File: c/sectomin.c
#include <stdio.h>
#include <stdlib.h>
#include <ctype.h>
#include <string.h>

int id(char *c) {
	for (int i=0;i<strlen(c);i++) {
		if (isdigit(c[i]) == 0) {
			return 0;
		}
	}
	return 1;
}

int main() {
	char t[10];
	printf("Input time in seconds: ");
	fgets(t, 10, stdin);
	fflush(stdin);
	t[strcspn(t, "\n")] = 0;
	if (! id(t)) {
		printf("No se ha ingresado un número\n");
		return 1;
	}
	int it = atoi(t);
	printf("%d:%d\n", it/60, it%60);
	return 0;
}

File: python/sectomin.py
# AUTHOR: Darth Venom
# CREATED: 16/3/2021
#
# ABOUT: Convierte segundos a minutos.
#
# DEPENDS ON: ---

def sectomin(t):
    if not t.isdigit:
        return
    t = int(t)
    return f"{int(t/60)}:{t%60}"

if __name__ == "__main__":
    t = input("Input time in seconds: ")
    response = sectomin(t)
    if response:
        print(response)
    else:
        print("No se ha ingresado un número")
        exit(1)
